src/routes/docs/products/functions/functions/+page.markdoc

```diff
@@ -178,3 +178,11 @@ You can access environment variables inside functions using your [runtime langua
 ## Timeout {% #timeout %}
 You can limit the execution time of your function by navigating to your function > **Settings** > **Timeout**.
 There is a system wide maximum timeout of 900 seconds (15 minutes).
+
+## Scopes {% #scopes %}
+
+You can configure the function permission scopes to access your project services, like users and buckets, by navigating to your function > **Settings** > **Scopes**.
```

Comment:
> function permission scopes 

This is for configuring the scopes for the dynamic API key. Now, if we mention dynamic API keys, how do we give the developer more info about what dynamic API keys are?

> project services

The scopes are not for accessing the service, but more so for accessing the resources.